doc/architecture/database_schemata.md

```diff
@@ -85,6 +85,70 @@ plant_detail }|--|| genus : ""
 plant_detail }|--|| subfamily : ""
 plant_detail }|--|| family : ""
 
+maps {
+  INT id PK
+  VARCHAR name "NOT NULL"
+  BOOLEAN is_version "NOT NULL"
+  VARCHAR version_name
+  BOOLEAN is_inactive "NOT NULL"
```

Comment:
Also make a date for inactivity, is more flexible (can be used to trigger deletion after some period) and consistent to deletion.